pyalgostrategypool/volatility_trend_atr_v2.py

```diff
@@ -0,0 +1,83 @@
+import talib
+from pyalgotrading.constants import *
+from pyalgotrading.strategy import StrategyBase
+
+
+class VolatilityTrendATRV2(StrategyBase):
+    name = 'Volatility Trend ATR V2'
```

Comment:
if there is no file called `Volatility Trend ATR`, then remove `V2` from here
if there is a file called `Volatility Trend ATR`, then paste this code there and remove this file. ofcourse, remove `V2` in this case too